dummyserver/testcase.py

```diff
@@ -163,18 +164,21 @@ class HTTPDummyServerTestCase:
 
     @classmethod
     def _start_server(cls) -> None:
-        cls.io_loop = ioloop.IOLoop.current()
-        app = web.Application([(r".*", TestingApp)])
-        cls.server, cls.port = run_tornado_app(
-            app, cls.io_loop, cls.certs, cls.scheme, cls.host
-        )
-        cls.server_thread = run_loop_in_thread(cls.io_loop)
+        with contextlib.ExitStack() as stack:
+            io_loop = stack.enter_context(run_loop_in_thread())
```

Comment:
What is the point of an `ExitStack` if you'll only have one `enter_context` call?